src/main/scala/com/localytics/sbt/sqs/ElasticMQPlugin.scala

```diff
@@ -0,0 +1,31 @@
+package com.localytics.sbt.sqs
+
+import com.localytics.sbt.sqs.ElasticMQKeys._
+import com.localytics.sbt.sqs.ElasticMQTasks._
+import sbt._
+
+object ElasticMQPlugin extends AutoPlugin {
+  
+  // auto enable plugin
+  override val trigger = allRequirements
+
+  // inject project keys
+  val autoImport = ElasticMQKeys
+
+  // inject defaults via projectSettings 
+  override lazy val projectSettings = Seq(
+
+    elasticMQVersion    := "0.9.0-beta1",
```

Comment:
The descriptions in the `Keys` say 0.8.12 is the default value. We should sync these two up.